File: TDCODER/Kỹ thuật lập trình 1/Chương 3 - Toán tử và biểu thức/KTLT1_B1C3.cs
using System;

public class Program {
    public static void Main(string[] args) {
        int a = int.Parse(Console.ReadLine());
        int b = int.Parse(Console.ReadLine());
        Console.Write("{0} {1} {2} {3:0.0}", a+b, a-b, a*b, (double) a/b);
    }
}

File: TDCODER/Kỹ thuật lập trình 1/Chương 7 - Mảng một chiều/KTLT1_B1C_3.cpp
using System;

public class Program {
    static void Main(string[] args) {
        int n = int.Parse(Console.ReadLine());
        int[] a = new int[n];
        for (int i = 0; i < n; i++)
            a[i] = int.Parse(Console.ReadLine());
        for (int i = 0; i < n; i++)
            if (a[i] % 2 == 0)
                Console.WriteLine(a[i]);
    }
}